lib/rucio/daemons/conveyor/poller.py

```diff
@@ -140,11 +140,22 @@ def poller(once=False, activities=None, sleep_time=60,
                 for transf in transfs:
                     if not transf['external_host'] in xfers_ids:
                         xfers_ids[transf['external_host']] = []
-                    xfers_ids[transf['external_host']].append((transf['external_id'], transf['request_id']))
+                    xfers_ids[transf['external_host']].append((transf['external_id'], transf['request_id'], transf['scope']))
```

Comment:
I  would vote to switch using a namedtuple here. This will require some refactoring, but will make the code better.